Makefile

```diff
@@ -0,0 +1,78 @@
+BASE_DOMAIN=origin-gce.dev.openshift.com
+MOUNT_FLAGS=:z
+PODMAN=sudo podman
+PODMAN_RUN=${PODMAN} run --privileged --rm -v `pwd`:/output${MOUNT_FLAGS} --user `id -u`
+PODMAN_PARAMS=-e OPENSHIFT_INSTALL_PLATFORM=libvirt \
+-e OPENSHIFT_INSTALL_LIBVIRT_URI=qemu+tcp://192.168.122.1/system \
+-e OPENSHIFT_INSTALL_CLUSTER_NAME=${USERNAME} \
+-e OPENSHIFT_INSTALL_BASE_DOMAIN=${BASE_DOMAIN} \
```

Comment:
If I don't include `-e OPENSHIFT_INSTALL_LIBVIRT_IMAGE="file:///unused" \` I get `dial tcp: i/o timeout` on the create install-config step.

Reply:
Fixed

Reply:
I wonder why I don't get an error when I leave off LIBVIRT_IMAGE.

Reply:
Me too. The error message is network related, and I've had to modify the podman flags in the past to use my host network rather than the bridge due to our corporate firewall, so the firewall could be a difference. But I really don't know.